packages/block-library/src/nextpage/block.json

```diff
@@ -1,9 +1,6 @@
 {
 	"name": "core/nextpage",
 	"category": "layout",
-	"parent": [
-		"core/post-content"
-	],
```

Comment:
@epiqueras there might be a better fix here that keeps the parent property too but it's better to allow page break anywhere then to confuse all users with this block showing up first.

Reply:
I opened this https://github.com/WordPress/gutenberg/pull/22523 as an alternative (better) fix that will allow us to restore this.